Gems/GradientSignal/Code/Include/GradientSignal/Components/GradientTransformComponent.h

```diff
@@ -130,6 +131,9 @@ namespace GradientSignal
         AZ::Vector3 GetBounds() const override;
         void SetBounds(AZ::Vector3 bounds) override;
 
+        AZ::Vector3 GetCenter() const override;
+        void SetCenter(AZ::Vector3 center) override;
```

Comment:
The parameter in `SetCenter` is passed by value and therefore making copies. Any particular reason for not using const ref?

Same for `SetBounds`, `SetTranslate`, `SetRotate` and `SetScale`. It'd be nice to use const ref in all of them.

Reply:
I wondered about that too, but I just copied what all the other functions here were doing.

Reply:
ok, can do.

Reply:
updated

Reply:
Thanks!